testsuite/features/step_definitions/common_steps.rb

```diff
@@ -251,6 +251,11 @@
   step %(I select "#{label}" from "original_id")
 end
 
+When(/^I select the custom architecture channel for "(.*?)" as the origin channel$/) do |client|
+  label = deb_host?(client) ? "Test-Channel-Deb-AMD64 Child Channel" : "Test-Channel-x86_64 Child Channel"
```

Comment:
I would suggest to have a dict/map into constants.rb, so you can have a unique source of this info, and then you select it here with something like TEST_CHANNEL_BY_ARCH[node.architecture] after including an attribute architecture in our Labanda module.

Reply:
This step was created to anticipate the use of synced products for other architecture so right now it is single-use
However if we might use it for other architectures in the future it is a good idea to do that